SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -218,6 +220,11 @@ TArray<FWorkerComponentData> EntityFactory::CreateEntityComponents(USpatialActor
 	ComponentDatas.Add(NetOwningClientWorker(GetConnectionOwningWorkerId(Channel->Actor)).CreateNetOwningClientWorkerData());
 	ComponentDatas.Add(AuthorityIntent::CreateAuthorityIntentData(IntendedVirtualWorkerId));
 
+	if (!Actor->IsHidden())
```

Comment:
I believe we should also add the `bIsAlwaysRelevant` check here right? If the actor is spawned with `bHidden` and `bAlwaysRelevant` both true then we still want the actor to be visible.

Reply:
Now that I think about it, how is this playing with actors who spawn hidden like the game mode? Are they still replicating at the moment? Shouldn't we be special casing them here too when adding the visible component?